### DISCUSSION TITLE
请帮忙修改一下，使用这个模板后标题无法显示在首页的条目笔记下，那里是空白的

### DISCUSSION THREAD

**Discussion by 520max** [windingwind/zotero-better-notes#1074] [Q&A]
<html>
<hr/>
<table border="1">

<h1 style="color:  #E65100; background-color:  #FFF8E1;">(${topItem.getField("date")}) ${topItem.getField("title")}</h2>

<tr>
<td><b>期刊: <b style="color:#FF0000">${topItem.getField('publicationTitle')}</b></b>（发表日期: <b>${topItem.getField("date")}</b>）<br><b>作者:</b> ${topItem.getCreators().map((v)=>v.firstName+" "+v.lastName).join("; ")}</td>
</tr>

<tr>
<td><b>摘要: </b><i>${topItem.getField('abstractNote')}</i></td>
</tr>

    <tr><td>
        <b>期刊分区: </b>
        
        ${{
        let space = " ㅤㅤ ㅤㅤ"
        return Array.prototype.map.call(
          Zotero.ZoteroStyle.api.renderCell(topItem, "publicationTags").childNodes,
          e => {
            e.innerText =  space + e.innerText + space;
            return e.outerHTML
          }
          ).join(space)
        }}$
      </td></tr>

 
    <tr><td>
        ${(() => {
          const attachments = Zotero.Items.get(topItem.getAttachments());
          if (attachments && attachments.length > 0) {
            return `<b>原文pdf链接: </b><a href="zotero://open-pdf/0_${attachments[0].key}">${attachments[0].getFilename()}</a>`;
          } else {
            return `<b>原文pdf链接: </b>`;
          }
        })()}
      </td></tr>

<tr>
<td><b>笔记创建日期: </b>${new Date().toLocaleString()}</td>
</tr>

</table>

</span></h2>\n <h2 style="color:  #E65100; background-color:  #FFF8E1;">💡创新点</h2> <hr/>

<blockquote>Tips: 本文提出了什么<u>新的科学问题</u>，提出了什么<u>新的研究思路</u>，或提出了什么<u>新的研究工具</u>？</blockquote>
</span></h1>\n <h2 style="color:  #E65100; background-color:  #FFF8E1;">💡论文框架</h2> <hr/>
<p></p>

</span></h1>\n <h2 style="color:#2E7D32; background-color:    #F1F8E9;">📚一、研究背景</h2> <hr/>
<blockquote>Tips: 本研究的<u>必要性、紧迫性、可行性</u>是什么？作者是如何论述的？<br>作者引用了哪些该领域的<u>开创性文献</u>？</blockquote>
<h3 style="color:#2E7D32;">1. 为什么要研究这个问题？</h3>
<blockquote>论文研究了什么问题，为什么研究这个问题？</blockquote><p></p>
<h3 style="color:#2E7D32;">2. 学者们做了哪些研究来解决此问题？</h3>
<blockquote><作者在介绍中总结了“历史上学者们做了哪些研究来解决此问题”？请针对该论文的介绍中历史学者的工作进行总结，并以列表的形式输出。</blockquote><p></p>
<h3 style="color:#2E7D32;">3. 这些解决方法还有什么不足？</h3>
<blockquote>根据左侧网页中的论文的Introduction章节内容，针对论文作者所研究的问题，对论文作者所总结的其他学者在历史上进行的研究工作和所提出的解决方法的不足之处进行概括和总结，以列表的形式输出。</blockquote><p></p>
<h3 style="color:#2E7D32;">4. 论文提出了什么解决方法？</h3>
<blockquote>论文的作者提出了什么解决方法，解决了什么问题，获得了什么样的进步？请根据左侧网页中的论文全文进行总结，并采用列表的形式回答。</blockquote><p></p>
</span></h1>\n <h2 style="color:#4A148C; background-color:    #F5F5F5;">🧩二、研究方法</h2> <hr/>

<h3 style="color:#4A148C;">6. 论文采用了什么方法来达到自己的研究目的？</h3>
<blockquote>论文采用了什么方法来达到自己的研究目的？核心的方法内容（如公式、框图、算法等）是什么？</blockquote><p></p>
</span></h1>\n <h2 style="color: #006064; background-color:   #E0F7FA;">🔬三、重要结论</h2> <hr/>

<h3 style="color: #006064;">7. 论文的核心结论是什么？</h3><p></p>
<h3 style="color: #006064;">8. 论文的创新点是什么？</h3>
<blockquote>论文的创新点是什么？论文提出了新的理论还是在已有的理论上改进了研究方法，具体是什么？</blockquote><p></p>
</span></h1>\n <h2 style="color:#1565C0; background-color:   #E1F5FE;">📜四、创新&疑问</h2> <hr/>

<h3 style="color:#1565C0; ">9. 论文工作的不足之处有哪些？</h3>
<blockquote>论文工作的不足之处有哪些？论文提出的方法的限制是什么？</blockquote><p></p>
<h3 style="color:#1565C0; ">10.未来可以进一步在哪些方面完善论文的研究或弥补论文的不足？</h3><p></p>
</span></h1>\n <h2 style="color:#4A148C; background-color:    #F5F5F5;">🤔五、展望与收获</h2> <hr/>

<h3 style="color:#4A148C;">11.我们从论文中得到了什么收获，产生了什么idea？</h3><p></p>
<h3 style="color:#4A148C;">12.相关基础知识和专业名词？</h3><p></p>
<h3 style="color:#4A148C;">13.经典语句记录。</h3><p></p>
</html>

**Comment by windingwind** [comment]
笔记标题即第一行，与使用什么模板无关

**Reply by windingwind** [reply]
@520max 自己修改第一行内容即可。我也不知道你要什么第一行内容

**Comment by 520max** [comment]
![image](https://github.com/user-attachments/assets/a4bf14a0-b7cb-4a3a-8e5b-b54dc0cb42c0)

**Reply by 520max** [reply]
遇到一个问题是，在这个页面的笔记模版标题无法显示出来，是不是上面的代码有问题？
